CMSgov/mint-app
# [EASI-3725] - Update model statuses

# EASI-3725

## Changes and Description

- Updated status text per [Figma](https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?type=design&node-id=7497-25658&mode=design&t=yEdwiUlWAnorlQmm-0)
- Updated unit tests



## How to test this change

- Enter into a model plan's task list and click to update status
- Verify the list of statuses matches Figma

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
